pkg/manifests/manifests.go

```
@@ -62,3 +67,34 @@ func GetFileMultipleYamls(filename string, decoder DecodeFormat) ([]interface{},
 
 	return outputList, nil
 }
+
+func GetPullSecret() string {
+	var secret corev1.Secret
+	if err := GetFileData("pull-secret.yaml", &secret); err != nil {
+		logrus.Errorln("Error reading pull secret:", err.Error())
+		os.Exit(1)
```

This should return an error, since when it will be integrated within the installer it will be managed by the upper level command

---

I tend to agree - we will have to eventually pass errors all the way up, so while there are some places up the stack that we already don't return the error, any new function that just bails out is adding technical debt.

---

Ok. Error will be returned.